cmd/template/cluster/provider/capa.go

```diff
@@ -0,0 +1,62 @@
+package provider
```

Comment:
New file for `capa` and `capz` to de-clutter the old files. Otherwise it felt really messy to have two different implementations in one file.